sdk/azcore/internal/pollers/op/op.go

```diff
@@ -61,6 +61,7 @@ func New(resp *http.Response, pollerID string) (*Poller, error) {
 	// calculate the tentative final GET URL.
 	// can change if we receive a resourceLocation.
 	// it's ok for it to be empty in some cases.
+	// TODO: finalState
```

Comment:
Is there still a TODO item here?

Reply:
Yeah.  We need to add test coverage for data-plane pollers in autorest to ensure we get the algorithm right (comment applies to both TODOs).

Reply:
Tracked in https://github.com/Azure/azure-sdk-for-go/issues/17415